src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -387,7 +387,7 @@ protected function getEnv(string $name)
             $prefix = substr($name, 0, $i);
             $localName = substr($name, 1 + $i);
         } else {
-            $prefix = 'string';
+            $prefix = '';
```

Comment:
Damn, what if people override the default string prefix :thinking: This is probably not a good solution.